webpages/css/main.css

```diff
@@ -198,17 +207,26 @@ main.index > section > * {
 }
 
 main.index > section:nth-of-type(4n+2) {
-  background-color: #90e8ef; /* from poster */
+  /*background-color: #90e8ef; /* from poster */
+  /*background-color: #D6EDFE; /* from poster */
+  background-image: linear-gradient(to bottom, #D6EDFE, #DADAFE);
```

Comment:
not sure we like the red hue; how would you feel about this: have here what you have on line 228; and then change line 228 to go `to top` so the light bit is by the dark section.